### PR TITLE
Support new snippet atoms & initial server side changes

### PR DESCRIPTION
**Pull request by mchv**
## What does this change?

This adds the plumbing for 4 new atoms we are adding to the platform.

- [Content API models related changes](https://github.com/guardian/content-api-models/pull/112)
- [Atom maker related changes](https://github.com/guardian/atom-maker/pull/27)
- [Atom workshop related changes](https://github.com/guardian/atom-workshop/pull/132)
- [Elasticsearch related changes]()
- [Concierge & Porter related changes](https://github.com/guardian/content-api/pull/1928) 
## Does this affect other platforms - Amp, Apps, etc?

We are not releasing `amp` yet but we are going to have a look when we implement the `html` & `css` to support them properly.

## Screenshots

Not yet as this is only plumbing for the time being but this is how they will look like:

snippets_profile2
snippets_q_and_a2
snippets_quick_guide2
snippets_timeline2

### CODE REVIEW

File: common/app/model/content/Atoms.scala [hunk 1]
```diff
@@ -423,3 +463,20 @@ object ExplainerAtom {
     ExplainerAtom(atom.id, explainer.tags.getOrElse(Nil), explainer.title, explainer.body)
   }
 }
+
+object QandaAtom {
+  def make(atom: AtomApiAtom): QandaAtom = QandaAtom(atom.id, atom, atom.data.asInstanceOf[AtomData.Qa].qa)
```
Comment (tomrf1) [inline common/app/model/content/Atoms.scala]: this will need updating to qanda

**Comment by TBonnin** [comment]
I wonder if we could abstract all this repetitive boilerplate (macro?)

**Comment by LATaylor-guardian** [comment]
@TBonnin You could, but its whether it's worth the effort and you introduce something quite complex for what in fact should be relatively simple.  I think it's symptomatic of a wider problem to be honest that could be solved more simply. Am happy to discuss with you.

**Comment by tomrf1** [comment]
👍

**Pull request merged**